backend/functions.py

```diff
@@ -100,56 +106,75 @@ def download_avatar(user_id, username, user_url, output_folder):
                 image_url = meta_tag["content"]
                 response = get(image_url)
                 if response.status_code == 200:
-                    with open(avatar_filename, 'wb') as image_file:
+                    with open(avatar_filename, "wb") as image_file:
                         image_file.write(response.content)
                     print(f"Downloaded avatar for user {user_id}-{username} successfully")
                 else:
-                    print(f"Failed to download avatar for user {user_id}-{username}. Status code: {response.status_code}")
+                    print(
+                        f"Failed to download avatar for user {user_id}-{username}. Status code: {response.status_code}"
+                    )
             else:
                 print(f"No profile photo found for user {user_id}-{username}")
         else:
             print(f"Failed to fetch user page for user {user_id}-{username}. Status code: {response.status_code}")
     except Exception as e:
         print(f"Error downloading avatar for user {user_id}-{username}: {e}")
 
+
 def download_avatars(json_file, output_folder):
     print(f"Starting avatars download based on {json_file}...")
-    with open(json_file, 'r') as f:
+    with open(json_file) as f:
         data = load(f)
 
     with ThreadPoolExecutor(max_workers=5) as executor:
         for user_id, user_data in data.items():
             # Skip users without a photo_id
-            if user_data.get('photo_id') is None:
+            if user_data.get("photo_id") is None:
                 continue
-            username = user_data.get('username', '')
+            username = user_data.get("username", "")
             user_url = f"https://t.me/{username}"
             executor.submit(download_avatar, user_id, username, user_url, output_folder)
 
+
 def create_config(file_path):
     """Create a custom YAML configuration file by asking the user for input."""
 
     settings = {
-        'api_config': {
-            'phone': {'prompt': 'TG phone number: ', 'default': None, 'mandatory': True},
-            'api_id': {'prompt': 'TG api_id: ', 'default': None, 'mandatory': True, 'type': int},
-            'api_hash': {'prompt': 'TG api_hash:', 'default': None, 'mandatory': True},
+        "api_config": {
+            "phone": {"prompt": "TG phone number: ", "default": None, "mandatory": True},
+            "api_id": {"prompt": "TG api_id: ", "default": None, "mandatory": True, "type": int},
+            "api_hash": {"prompt": "TG api_hash:", "default": None, "mandatory": True},
+        },
+        "location": {
+            "lat": {"prompt": "Starting latitude [51.51404]: ", "default": 51.51404, "mandatory": False, "type": float},
+            "lon": {
+                "prompt": "Starting longitude [-0.15063]: ",
+                "default": -0.15063,
+                "mandatory": False,
+                "type": float,
+            },
```

Comment:
Since we do not check line length it is better to keep unified code style for this map.
I.e. each parameter on new line or keep existent definition.